src/compiler/transformers/utilities.ts

```diff
@@ -143,6 +143,7 @@ namespace ts {
             createLiteral(externalHelpersModuleNameText));
 
         if (externalHelpersImportDeclaration) {
+            externalHelpersImportDeclaration.transformFlags |= TransformFlags.NeverApplyImportHelper;

```

Comment:
We shouldn't set transform flags explicitly. See my related comment in _program.ts_.